crates/db_schema/src/utils.rs

```diff
@@ -59,6 +58,14 @@ pub const SITEMAP_LIMIT: i64 = 50000;
 pub const SITEMAP_DAYS: Option<TimeDelta> = TimeDelta::try_days(31);
 pub const RANK_DEFAULT: f64 = 0.0001;
 
+/// Change collapse limits from 8 to 11 so the query planner can find a better table join
+/// order for more complicated queries
+const CONNECTION_OPTIONS: [&str; 1] = [
+  "geqo_threshold=12",
+  // TODO these two config options cause really slow queries, not sure why they were added.
+  // "from_collapse_limit=11",
+  // "join_collapse_limit=11",
```

Comment:
Not sure when or why we added these settings, but they were causing lots of slowness.

Reply:
Just remove the options if they are not helping.

Reply:
Alright.